nncf/tensorflow/quantization/default_quantization.py

```diff
@@ -43,12 +43,24 @@
         layer_metatypes.TFPReLULayerMetatype,
         layer_metatypes.TFLeakyReLULayerMetatype,
         layer_metatypes.TFActivationLayerMetatype,
+        # op_metatypes.TFAvgPoolOpMetatype,
+        # op_metatypes.TFAvgPool3DOpMetatype,
+        # op_metatypes.TFMeanOpMetatype,
+        # op_metatypes.TFResizeNearestNeighborOpMetatype,
+        # op_metatypes.TFEluOpMetatype,
+        # op_metatypes.TFLeakyReluOpMetatype,
+        # op_metatypes.TFRelu6OpMetatype,
+        # op_metatypes.TFReluOpMetatype,
```

Comment:
Commented code is not allowed, same for commented code below

Reply:
I fixed it.